pootle/apps/pootle_store/receivers.py

```diff
@@ -0,0 +1,10 @@
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from pootle_store.models import Unit
+
+
+@receiver(post_save, sender=Unit)
+def unit_priority_handler(sender, **kwargs):
```

Comment:
Since this only makes sense when `virtualfolder` app is enabled, what about moving this handler to that app?